modules/vm/variables.tf

```diff
@@ -139,13 +139,11 @@ variable "subnet_id" {
 variable "infrastructure_admin_ssh_public_key_filepath" {
   description = "Public ssh key file path."
   type = string
-  default = "~/.ssh/id_rsa.pub"
```

Comment:
I think we should keep the default value for keys, and give user a way to provide another key through -var if they want to

Reply:
removing default will just cause more prompts for the user to fill while doing `terraform apply`, which could be annoying for the user at times

Reply:
defaults exists in the root variables.tf, no need to provide default again inside vm module

Reply:
The default is in the top level variables.tf.  Does it hurt to leave it here too?

Reply:
I need to check that